python/ray/data/read_api.py

```diff
@@ -673,6 +680,8 @@ def read_parquet(
         partition_filter: A
             :class:`~ray.data.datasource.partitioning.PathPartitionFilter`. Use
             with a custom callback to read only selected partitions of a dataset.
+        shuffle: If setting to "files", randomly shuffle input files order before read.
+            Defaults to not shuffle with ``None``.
```

Comment:
it's a bit weird  to use `shuffle="files"`, while we actually shuffle the fragments. I don't really oppose this, but at least we should make this docstring accurate.

Reply:
Alternatively, we could do `shuffle="fragments"` for Parquet